lib/config.js

```diff
@@ -13,6 +13,15 @@ function camelify(res) {
   }, {});
 }
 
+function isEnvVarContainingPassword(envVarName) {
+  const enVarsContainingPasswords = [
+    'PASSWORD',
+    'BROKER_CLIENT_VALIDATION_AUTHORIZATION_HEADER',
+    'BROKER_CLIENT_VALIDATION_BASIC_AUTH',
+  ];
```

Comment:
The Broker code doesn't know anything about SCMs or their env vars. I think it should stay that way

Reply:
Maybe the solution is to have an env var with the property names that contain secrets.

`ENV_VARS_CONTAINING_PASSWORDS="BROKER_CLIENT_VALIDATION_AUTHORIZATION_HEADER, BROKER_CLIENT_VALIDATION_BASIC_AUTH"`